sonic-net/sonic-buildimage
# [Mellanox] CMIS management feature enablement

Depends on https://github.com/sonic-net/sonic-buildimage/pull/18571


#### Why I did it
CMIS management feature enablement 

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
Add new variable with value 1 for CMIS management feature enablement under specific HWSKU folder. Not the default one

#### How to verify it
Install image on switch with same HWSKU and check "show interfaces status" (Type column)



#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
